command/riff_invoker.go

```diff
@@ -115,9 +117,16 @@ func NewCommandInvoker(build libjavabuildpack.Build) (RiffCommandInvoker, bool,
 	return RiffCommandInvoker{
 		application:   build.Application,
 		executable:    exec,
-		launch:        build.Launch,
-		invokerLayer:  build.Launch.DependencyLayer(dep),
-		functionLayer: build.Launch.Layer("function"),
+		layers:        build.Layers,
+		invokerLayer:  build.Layers.DependencyLayer(dep),
+		functionLayer: build.Layers.Layer("function"),
 	}, true, nil
+}
+
+type marker struct {
```

Comment:
I think this also needs to contain the language of the function, in the weird case where you would 
- change the language of the function
- keep the same artifact
- the value of the FUNCTION_URI should not be the same
(very very unlikely I admit, but...)

Reply:
Yep, totally legit.